src/clusterfuzz/_internal/protos/uworker_msg.proto

```diff
@@ -86,6 +86,7 @@ message Input {
   optional RegressionTaskInput regression_task_input = 17;
   optional SymbolizeTaskInput symbolize_task_input = 18;
   optional string module_name = 19;
+  map<string, string> progression_uworker_env = 20;  
```

Comment:
This would eventually replace `uworker_env`, right? The naming is just because this PR only touches `progression_task` so far?

Reply:
Indeed. I also think that using protos directly would force us to actively try to simplify the types used and avoid using the user-defined JSON proto msg.

Reply:
Can you mention that in a comment please?

Reply:
I think there's some code in postprocess or error handling that grabs values from uworker_input.uworker_env. We could change that to grab directly from the env (though this will make testing harder).